briefcase/app.py

```diff
@@ -52,6 +52,14 @@ class app(Command):
          'URL for the support package to use'),
         ('download-dir=', None,
          "Directory where the project support packages will be cached"),
+        ('build', None,
+         "Build the project after generating"),
+        ('execute', None,
+         "Run the application after building"),
+        ('os-version=', None,
+         "For iOS, the OS version to run (e.g., iOS 10.2)"),
+        ('device=', None,
+         "For iOS, the device to run (e.g., iPhone 7 Plus)")
```

Comment:
See above, re: device. Let's generalize, and leave ourselves open for the future.